src/Components/Common/Header/index.tsx

```diff
@@ -1,14 +1,15 @@
 import * as S from "./style";
 import * as I from "../../../Asset/SVG/index";
 import { Link, useNavigate } from "react-router-dom";
-import { useRecoilState } from "recoil";
-import { HasAdminToken } from "Atom";
+import { useRecoilState, useRecoilValue } from "recoil";
+import { HasAdminToken, ShowBanner } from "Atom";
 import { useEffect, useState } from "react";
 import { faBars, faChevronUp } from "@fortawesome/free-solid-svg-icons";
```

Comment:
이런식이면 아무도 무슨 svg 키워드인줄 모르므로 as를 사용하여 다른 개발자들이 알기 쉽게 import 해오는것이 좋겠네요

Reply:
`<FontAwesomeIcon icon={faBars} />` 이런 형식으로 사용하는 거라서 필요 없을 줄 알았습니다
import 할 때 `import * as SVG from "@fortawesome/free-solid-svg-icons";` 이런 형식으로 할까요?

Reply:
```suggestion
import { Bar as faBars, Arrow as faChevronUp } from "@fortawesome/free-solid-svg-icons";
```
이런식을 말씀드린겁니다.

`faChevronUp` 이 친구는 뭔지를 모르겠네요.